src/routes/v2/teams.rs

```diff
@@ -65,6 +64,69 @@ pub async fn team_members_get_project(
 
         let user_id = current_user.as_ref().map(|x| x.id.into());
 
+        let logged_in = current_user
+            .and_then(|user| {
+                members_data
+                    .iter()
+                    .find(|x| x.user_id == user.id.into() && x.accepted)
+            })
+            .is_some();
+        let team_members: Vec<_> = members_data
+            .into_iter()
+            .filter(|x| {
+                logged_in
+                    || x.accepted
+                    || user_id
+                        .map(|y: crate::database::models::UserId| y == x.user_id)
+                        .unwrap_or(false)
+            })
+            .flat_map(|data| {
+                users.iter().find(|x| x.id == data.user_id).map(|user| {
+                    crate::models::teams::TeamMember::from(data, user.clone(), !logged_in)
+                })
+            })
+            .collect();
+        Ok(HttpResponse::Ok().json(team_members))
+    } else {
+        Ok(HttpResponse::NotFound().body(""))
+    }
+}
+
+#[get("{id}/members")]
+pub async fn team_members_get_organization(
+    req: HttpRequest,
+    info: web::Path<(String,)>,
+    pool: web::Data<PgPool>,
+    redis: web::Data<deadpool_redis::Pool>,
+    session_queue: web::Data<AuthQueue>,
+) -> Result<HttpResponse, ApiError> {
+    let string = info.into_inner().0;
+    let organization_data =
+        crate::database::models::Organization::get(&string, &**pool, &redis).await?;
+
+    if let Some(organization) = organization_data {
+        let current_user = get_user_from_headers(
+            &req,
+            &**pool,
+            &redis,
+            &session_queue,
+            Some(&[Scopes::ORGANIZATION_READ]),
+        )
+        .await
+        .map(|x| x.1)
+        .ok();
+
+        let members_data =
+            TeamMember::get_from_team_full(organization.team_id, &**pool, &redis).await?;
+        let users = crate::database::models::User::get_many_ids(
+            &members_data.iter().map(|x| x.user_id).collect::<Vec<_>>(),
+            &**pool,
+            &redis,
+        )
+        .await?;
+
+        let user_id = current_user.as_ref().map(|x| x.id.into());
+
```

Comment:
i dont think this route is needed given you already include it in the org reponse

Reply:
Yeah but removing it would remove the ability to check members for project teams as well, as this is a team route. 
I can remove its organization-specific functionality and keep the project one, but I think that might confuse users more ("if orgs use teams, why dont team routes work for them)